sql/moz-fx-data-shared-prod/fenix_derived/google_ads_campaign_cost_breakdowns_v1/query.sql

```diff
@@ -0,0 +1,79 @@
+WITH campaigns AS (
+  SELECT DISTINCT
```

Comment:
FYI in writing the conversions table, we realized the grain of `campaign_history` is start_date-id, so I had to date-join to get the campaign name (i.e. a single ID can have different names over time): https://github.com/mozilla/bigquery-etl/blob/main/sql/moz-fx-data-shared-prod/google_ads_derived/campaign_conversions_by_date_v1/query.sql

Reply:
ayayay